{"name": "Проверка невозможности создания курьера c существующим логином", "status": "failed", "statusDetails": {"message": "AssertionError: assert 'Этот логин у...буйте другой.' == 'Этот логин уже используется'\n  - Этот логин уже используется\n  + Этот логин уже используется. Попробуйте другой.", "trace": "self = <test_create_courier.TestCreateCourier object at 0x00000219DECB4790>, get_url = 'https://qa-scooter.praktikum-services.ru/api/v1/courier'\nget_new_data = {'firstName': 'mqlxahcxtk', 'login': 'otojjgltvs', 'password': 'itsrnoecbg'}\n\n    @allure.title('Проверка невозможности создания курьера c существующим логином')\n    def test_create_courier_with_exist_login_unavailable_success(self, get_url, get_new_data):\n        payload = exist_courier_data(get_new_data)\n        password = get_new_data['password']\n        payload['password'] = f\"new{get_new_data['password']}\"\n        response = get_response_post_courier(get_url, payload)\n        payload['password'] = password\n        message = response.json()['message']\n        assert response.status_code == HTTPStatus.CONFLICT\n>       assert message == Constants.ERROR_MESSAGE_FOR_EXIST_LOGIN\nE       AssertionError: assert 'Этот логин у...буйте другой.' == 'Этот логин уже используется'\nE         - Этот логин уже используется\nE         + Этот логин уже используется. Попробуйте другой.\n\ntests\\test_create_courier.py:72: AssertionError"}, "steps": [{"name": "Регистрация нового курьера и формирование тела запроса", "status": "passed", "parameters": [{"name": "get_new_data", "value": "{'login': 'otojjgltvs', 'password': 'itsrnoecbg', 'firstName': 'mqlxahcxtk'}"}], "start": 1706963796531, "stop": 1706963796727}, {"name": "Получение ответа на POST-запрос", "status": "passed", "parameters": [{"name": "get_url", "value": "'https://qa-scooter.praktikum-services.ru/api/v1/courier'"}, {"name": "payload", "value": "{'login': 'otojjgltvs', 'password': 'newitsrnoecbg', 'firstName': 'mqlxahcxtk'}"}], "start": 1706963796727, "stop": 1706963796950}], "start": 1706963796531, "stop": 1706963796953, "uuid": "ae8a9e6b-d8f7-41f6-98db-0ee1e557eaec", "historyId": "788a8f740c578dd029f3f2748cfba7b4", "testCaseId": "788a8f740c578dd029f3f2748cfba7b4", "fullName": "tests.test_create_courier#test_create_courier_with_exist_login_unavailable_success", "labels": [{"name": "tag", "value": "get_url('create_courier')"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_create_courier"}, {"name": "subSuite", "value": "TestCreateCourier"}, {"name": "host", "value": "DESKTOP-0EOUJEC"}, {"name": "thread", "value": "10908-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_create_courier"}]}